src/components/menu/MenuItem.tsx

```diff
@@ -46,9 +17,15 @@ export const MenuItem = ({
   className,
   icon,
   onClick,
+  style,
 }: MenuItemProps) => (
-  <StyledListItem className={className} onClick={onClick}>
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
+  <li
+    className={clsx(className, styles.styledListItem)}
+    onClick={onClick}
+    style={style}
+  >
```

Comment:
You may remove the eslint disable here too if adding`role="button"` to the element
```suggestion
  // eslint-disable-next-line jsx-a11y/click-events-have-key-events
  <li
    role="button"
    className={clsx(className, styles.styledListItem)}
    onClick={onClick}
    style={style}
  >
```